web_widget_timepicker/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
I didn't understand Pedro's comment in a PR before until I looked at the current README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst